docs/install.md

```diff
@@ -2,119 +2,224 @@
+4. You must have an accessible docker-registry (such as [Docker Hub](https://hub.docker.com)). The registry's URL and credentials are consumed by the applications via a pre-created secret
 
-```sh
-SHARED_DIR=~/mlrun-data
+> **Note:** These instructions use `mlrun` as the namespace (`-n` parameter). You may want to choose a different namespace in your kubernetes cluster.
 
-docker pull mlrun/jupyter:0.5.4
-docker pull mlrun/mlrun-ui:0.5.4
+### Chart Details
 
-docker network create mlrun-network
-docker run -it -p 8080:8080 -p 8888:8888 --rm -d --network mlrun-network --name jupyter -v ${SHARED_DIR}:/home/jovyan/data mlrun/jupyter:0.5.4
-docker run -it -p 4000:80 --rm -d --network mlrun-network --name mlrun-ui -e MLRUN_API_PROXY_URL=http://jupyter:8080 mlrun/mlrun-ui:0.5.4
+The MLRun Kit chart includes the following stack:
+
+- MLRun: <https://github.com/mlrun/mlrun>
+- Nuclio: <https://github.com/nuclio/nuclio>
+- Jupyter: <https://github.com/jupyterlab/jupyterlab> (+MLRun integrated)
+- NFS: <https://github.com/kubernetes-retired/external-storage/tree/master/nfs>
+- MPI Operator: <https://github.com/kubeflow/mpi-operator>
+
+### Installing the Chart
+
+Create a namespace for the deployed components:
+
+```bash
+kubectl create namespace mlrun
 ```
 
-When the execution completes &mdash;
+Add the `v3io-stable` helm chart repo
 
-- Open Jupyter Notebook on port 8888 and run the code in the [**examples/mlrun_basics.ipynb**](https://github.com/mlrun/mlrun/blob/master/examples/mlrun_basics.ipynb) notebook.
-- Use the MLRun dashboard on port 4000.
+```bash
+helm repo add v3io-stable https://v3io.github.io/helm-charts/stable
+```
 
-<a id="k8s-cluster"></a>
-## Install MLRun on a Kubernetes Cluster
+Create a secret with your docker-registry named `registry-credentials`:
 
-Perform the following steps to install and run MLRun on a Kubernetes cluster.
-> **Note:** The outlined procedure allows using the local, job, and Dask runtimes.
-> To use the MPIJob (Horovod) or Spark runtimes, you need to install additional custom resource definitions (CRDs).
+```bash
+kubectl --namespace mlrun create secret docker-registry registry-credentials \
+    --docker-server <your-registry-server> \
+    --docker-username <your-username> \
+    --docker-password <your-password> \
+    --docker-email <your-email>
+```
 
-- [Create a namespace](#k8s-create-a-namespace)
-- [Install a shared volume storage](#k8s-install-a-shared-volume-storage)
-- [Install the MLRun API and dashboard (UI) services](#k8s-install-mlrun-api-n-ui-services)
+where:
 
-<a id=k8s-create-a-namespace></a>
-### Create a namespace
+- `<your-registry-server>` is your Private Docker Registry FQDN. (https://index.docker.io/v1/ for Docker Hub).
+- `<your-username>` is your Docker username.
+- `<your-password>` is your Docker password.
+- `<your-email>` is your Docker email.
 
-Create a namespace for mlrun. For example:
+To install the chart with the release name `my-mlrun` use the following command.
```

Comment:
I would have rename `my-mlrun` to `mlrun-kit` 
no functional change just feels more professional IMO
this is just a suggestion, you don't have to take it

Reply:
done